tests/Service/FileUploaderTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Tests\Service;
+
+use App\Service\FileUploader;
+use Gaufrette\FilesystemInterface;
+use Symfony\Component\HttpFoundation\File\File;
+use Prophecy\Argument;
+use PHPUnit\Framework\TestCase;
+
+class FileUploaderTest extends TestCase
+{
+    private $filesystemProphecy;
+    private $fileUploader;
+
+    public function setUp()
+    {
+        $this->filesystemProphecy = $this->prophesize(FilesystemInterface::class);
+
+        $this->fileUploader = new FileUploader(
+            $this->filesystemProphecy->reveal()
+        );
+    }
+
+    public function testFileIsUploaded()
+    {
+        $this->filesystemProphecy->write(Argument::any(), Argument::any())->shouldBeCalled();
+
+        $this->fileUploader->upload(new File(__DIR__.'/../file/testFile', false));
```

Comment:
Not sure about the path here, I'd rather have any moving part in `var/`.

Reply:
After discussion, rather than moving to var/ it's been decided to move tests to `tests/src` and files required during tests (non auto generated/made to change) in `tests/resources`